examples/ex1.cpp

```diff
@@ -51,17 +51,20 @@
 #include "mfem.hpp"
 #include <fstream>
 #include <iostream>
+#include <chrono>
+#include <ctime>
 
 using namespace std;
 using namespace mfem;
 
 int main(int argc, char *argv[])
 {
    // 1. Parse command-line options.
-   const char *mesh_file = "../data/star.mesh";
+   const char *mesh_file = "../data/beam-hex.mesh";
+   //const char *mesh_file = "../data/star.mesh";
```

Comment:
I've been doing some performance runs with this example, I can revert the changes before we merge.